releases/release-5.4.2.md

```diff
@@ -0,0 +1,107 @@
+---
+title: TiDB 5.4.2 Release Notes
+---
+
+# TiDB 5.4.2 Release Notes
+
+Release Date: xx xx, 2022
+
+TiDB version: 5.4.2
+
+## __unsorted
+
++ PingCAP/TiDB
+
+    <!--transaction-->
+    - ```release-note [#34906](https://github.com/pingcap/tidb/issues/34906)
+    - ```release-note [#35198](https://github.com/pingcap/tidb/issues/35198)
+
+    <!--diagnosis-->
+    - ```release-note [#35340](https://github.com/pingcap/tidb/issues/35340)
+
+    <!--sql-infra-->
+    - ```release-note [#34722](https://github.com/pingcap/tidb/issues/34722)
+    - ```release-note [#34638](https://github.com/pingcap/tidb/pull/34638)
+    - ```release-note [#33965](https://github.com/pingcap/tidb/issues/33965)
+
+    <!--planner-->
+    - ```release-note [#34613](https://github.com/pingcap/tidb/issues/34613)
+
++ TiKV/TiKV
+
+    - Fix the issue of unexpected `panic` on analyzed statistics when `max_sample_size` is set to `0`. [#11192](https://github.com/tikv/tikv/issues/11192)
+    (dup: release-6.0.0-dmr.md > Bug fixes> TiKV)- Fix the potential issue of mistakenly reporting TiKV panics when exiting TiKV [#12231](https://github.com/tikv/tikv/issues/12231)
+    - Fix possible panic when source peer catch up logs by snapshot in merge [#12663](https://github.com/tikv/tikv/issues/12663)
+    - Fix potential panic when a peer is being split and destroyed at the same time [#12825](https://github.com/tikv/tikv/issues/12825)
+    - Reload TLS certificate automatically when it changes. [#12546](https://github.com/tikv/tikv/issues/12546)
+    - Fix bug which causes frequent pd client reconnection [#12345](https://github.com/tikv/tikv/issues/12345)
+    (dup: release-6.1.0.md > Improvements> TiKV)- Improve the health check to detect unavailable Raftstore, so that the TiKV client can update Region Cache in time [#12398](https://github.com/tikv/tikv/issues/12398)
```

Comment:
```suggestion
    - Improve the health check to detect unavailable Raftstore, so that the TiKV client can update Region Cache in time [#12398](https://github.com/tikv/tikv/issues/12398)
```